scilpy/utils/bvec_bval_tools.py

```diff
@@ -199,3 +199,26 @@ def mrtrix2fsl(mrtrix_filename, fsl_bval_filename=None,
                             filename_bval=fsl_bval_filename,
                             filename_bvec=fsl_bvec_filename,
                             verbose=1)
+
+
+def identify_shells(bvals, threshold=40.0):
+    """See DIPY's PR, will be added."""
```

Comment:
We should have a real docstring until the DIPY version is merged, to make sure we understand what is going on (and in case it isn't actually merged in DIPY)

Reply:
Is that clear?